pandas/core/computation/expressions.py

```diff
@@ -80,7 +80,7 @@ def _can_use_numexpr(op, op_str, a, b, dtype_check):
             # check for dtype compatibility
             dtypes = set()
             for o in [a, b]:
-                if hasattr(o, "dtypes"):
+                if hasattr(o, "dtypes") and o.ndim > 1:
```

Comment:
Is this actually needed?

Reply:
Ah sorry didn't realize that `dtypes` was implemented on a Series, so makes sense

Reply:
This may be worth adding a comment for. I also didn't realize that Series.dtypes was a thing.

Reply:
can we get here with anything other than DataFrame/Series?  if not, can this just be an `ndim` check instead of a hasattr check?

Reply:
You can get ndarray as well which doesn't have dtypes.